src/uwtools/resources/jsonschema/esg-grid.jsonschema

```diff
@@ -0,0 +1,84 @@
+{
+  "properties": {
+    "esg_grid": {
+      "additionalProperties": false,
+      "properties": {
+        "execution": {
+          "$ref": "urn:uwtools:execution"
+        },
+        "namelist": {
+          "additionalProperties": false,
+          "anyOf": [
+            {
+              "required": [
+                "base_file"
+              ]
+            },
+            {
+              "required": [
+                "update_values"
+              ]
+            }
+          ],
+          "properties": {
+            "base_file": {
+              "type": "string"
+            },
+            "update_values": {
+              "properties": {
+                "regional_grid_nml": {
+                  "additionalProperties": {
+                    "type": [
+                      "array",
+                      "boolean",
+                      "number",
+                      "string"
+                    ]
+                  },
```

Comment:
```suggestion
                  "additionalProperties": false
```
The notes doc indicated that we didn't think we'd need additional properties for this component. If not, we could simplify this.